### ISSUE TITLE
Tooltip is not provided for 'More' control present on the page.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1037]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1240881

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre-Requisite:
Select "Core(SQL)" in API combo box while creating the Cosmos DB.

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate through all the controls under SQL API Section.
Navigate to More controls present with expand collapse control.
Press F12 key to open the Developers Tool and inspect the code.
Verify the issue.

Actual Result:
Tooltip is not provided for 'More' control present on the page.

Expected Result:
Tooltip should be provided for 'More' control present on the page.


Note:
Same issue exists for all more controls present on the page under 'SQL API' section.


![image](https://user-images.githubusercontent.com/8519407/132718901-bdfb987d-5bf5-482a-a02e-13686ecfcc8f.png)
